integration/crds/lightweight/DanmNet.yaml

```diff
@@ -1,116 +1,126 @@
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              AllowedTenants:
+                items:
                   type: string
-                vxlan:
-                  type: integer
-                  format: int32
-                  minimum: 1
-                  maximum: 16777214
-                vlan:
-                  type: integer
-                  format: int32
-                  minimum: 1
-                  maximum: 4094
-                rt_tables:
-                  type: integer
-                  format: int32
-                  minimum: 0
-                  maximum: 255
-                net6:
-                  oneOf:
-                  - type: string
-                    maxLength: 0
-                  - type: string
-                    format: cidr
-                    pattern: ':'
-                allocation_pool_v6:
-                  properties:
-                    start:
-                      oneOf:
-                      - type: string
-                        maxLength: 0
-                      - type: string
-                        format: ipv6
-                    end:
-                      oneOf:
-                      - type: string
-                        maxLength: 0
-                      - type: string
-                        format: ipv6
-                    lastIp:
-                      oneOf:
-                      - type: string
-                        maxLength: 0
-                      - type: string
-                        format: cidr
-                        pattern: ':'
-                    cidr:
-                      oneOf:
-                      - type: string
-                        maxLength: 0
-                      - type: string
-                        format: cidr
-                        pattern: ':'
-                routes:
-                  type: object
-                routes6:
-                  type: object
+                type: array
+              NetworkID:
+                type: string
+              NetworkType:
+                type: string
+              Options:
+                properties:
+                  alloc:
+                    description: bit array of tracking address allocation
+                    type: string
+                  alloc6:
+                    description: bit array tracking IPv6 allocations
+                    type: string
+                  allocation_pool:
+                    description: subset of the IPv4 subnet from which IPs can be allocated
+                    properties:
+                      end:
```

Comment:
all the semantic checks like IP regex and integer range requirements were cut, any particular reason for this? the API semantics should not be changed, just updated